src/middlewares/Checking.js

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect} from 'react'
+import {useHistory} from 'react-router-dom'
+
+const Checking = () => {
+
+  const history = useHistory();
+
```

Comment:
useHistory is deprecated please change it to useNavigate.

Reply:
I think there is might updates in react-router-dom. useHistory is acclaimed and useNaviagte is deprecated. Please do notify me if there is something i missed.

Reply:
useHistory was deprecated in 2019.In the latest version of react router dom useNavigate is better option to use.

Reply:
Yes, I have used useNavigate.